utils/emailSender.ts

```diff
@@ -0,0 +1,36 @@
+import nodemailer from "nodemailer";
+require("dotenv").config();
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: parseInt(process.env.SMTP_PORT as string, 10),
+  secure: false,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASS,
+  },
+});
+
+export const sendEmail = async (
+  from: string,
```

Comment:
We want to be careful around things surrounding email since it's easy to abuse. 
Why is this first variable named `from` when the variable we're sending here is `EMAIL_TO`? The naming here is confusing, can we straighten that out?